spec/unit/provider/package/gem_spec.rb

```diff
@@ -165,6 +165,16 @@
         ])
       end
 
+      it "should not list 'default: ' text from rubygems''" do
+        provider_class.expects(:execute).with(%w{/my/gem list --local}, {:failonfail => true, :combine => true, :custom_environment => {"HOME"=>ENV["HOME"]}}).returns <<-HEREDOC.gsub(/        /, '')
+        bundler (1.16.1, default: 1.16.0, 1.15.1)
```

Comment:
if you don't care about the actual options being passed, e.g. `failonfail`, etc, you can do `expects(:execute).with(%w{/my/gem list --local}, anything).returns(...)`. The `anything` is a [mocha parameter matcher](http://gofreerange.com/mocha/docs/Mocha/ParameterMatchers.html) that matches anything you pass it.